displayer.go

```diff
@@ -0,0 +1,7 @@
+package plugin
+
+// Displayer is the interface plugins should implement to show a text to the user. The text will appear on the plugin details page and can be multi-line. Good for logging and providing webhook URLs to users.
+type Displayer interface {
+	Plugin
+	GetDisplay() string
```

Comment:
We should probably support markdown, could you add that GetDisplay can return a markdown string? https://www.npmjs.com/package/react-markdown

Reply:
I think we need to first discuss what the information is mainly for:
If it is just for displaying status, logs, etc. I would choose not to support markdown.

If it is to generate an instruction(eg: a netlify webhook plugin can dynamically generate a setup instruction from the current configuration and user context(setting up the webhook on netlify), then it would be great to support markdown)

Reply:
I'd say it's for showing instructions to the user.

Reply:
Okay, I will have this added